src/main/scala/mesosphere/marathon/core/task/termination/KillService.scala

```diff
@@ -22,16 +23,6 @@ trait KillService {
     */
   def killInstances(instances: Seq[Instance], reason: KillReason): Future[Done]
```

Comment:
Why is this still there when we remove `killInstance`?

Reply:
good catch, removed